README.md

```diff
@@ -27,41 +58,19 @@ Example:
 - [Windows](./INSTALL.md#windows-installation)
 - [Docker](./INSTALL.md#docker-installation)
 
-
 3. Refer to these [instructions](./DEVELOPMENT.md) for making code changes.
 
-If you intend to use FHIR Implementation Guides read the [Using Implementation Guides](./USING_IMPLEMENTATION_GUIDES.md) documentation first. 
-
-If you intend to do a multi-tenant deployment read the [Using Multi-Tenancy](./USING_MULTI_TENANCY.md) documentation first. 
-
-## Architecture
-
-The system architecture consists of multiple layers of AWS serverless services. The endpoint is hosted using API Gateway. The database and storage layer consists of Amazon DynamoDB and S3, with Elasticsearch as the search index for the data written to DynamoDB. The endpoint is secured by API keys and Cognito for user-level authentication and user-group authorization. The diagram below shows the FHIR server’s system architecture components and how they are related.
- 
-![Architecture](resources/architecture.png)
-
-## Components overview
+If you intend to use FHIR Implementation Guides, read the [Using Implementation Guides](./USING_IMPLEMENTATION_GUIDES.md) documentation first. 
 
-FHIR Works on AWS is powered by single-function components. These functions provide you the flexibility to plug your own implementations, if needed. The components used in this deployment are:
-+ [Interface](https://github.com/awslabs/fhir-works-on-aws-interface) - Defines communication between the components.
-+ [Routing](https://github.com/awslabs/fhir-works-on-aws-routing) - Accepts HTTP FHIR requests, routes it to the other components, logs the errors, transforms output to HTTP responses and generates the [Capability Statement](https://www.hl7.org/fhir/capabilitystatement.html).
-+ [Authorization](https://github.com/awslabs/fhir-works-on-aws-authz-rbac) - Accepts the access token found in HTTP header and the action the request is trying to perform. It then determines if that action is permitted.
-+ [Persistence](https://github.com/awslabs/fhir-works-on-aws-persistence-ddb) - Contains the business logic for creating, reading, updating, and deleting the FHIR record from the database. FHIR also supports ‘conditional’ CRUD actions and patching. 
-   + Bundle - Supports multiple incoming requests as one request. Think of someone wanting to create five patients at once instead of five individual function calls. There are two types of bundles, batch and transaction. We currently only support transaction bundles.
-+ [Search](https://github.com/awslabs/fhir-works-on-aws-search-es) - Enables system-wide searching (/?name=bob) and type searching (/Patient/?name=bob).
-+ History - (*Not implemented*) Searches all archived/older versioned resources. This can be done at a system, type or instance level.
-
-## License
-
-This project is licensed under the Apache-2.0 license.
+If you intend to do a multi-tenant deployment, read the [Using Multi-Tenancy](./USING_MULTI_TENANCY.md) documentation first. 
```

Comment:
For information about multi-tenant deployment, see Using multi-tenancy

Ref: https://alpha-docs-aws.amazon.com/awsstyleguide/latest/styleguide/crossreferences.html